service-api/module/Application/src/Yoti/SessionStatusService.php

```diff
@@ -101,7 +101,14 @@ private function getDocumentScanned(string $mediaId, string $yotiSessionId): arr
     private function evaluateFinalResult(array $checks, ?string $mediaId, CaseData $caseData): bool
     {
         //If UK passport ensure document presented was in date range
-        if (is_string($mediaId) && $caseData->idMethod === "po_ukp") {
+        /**
+         * @psalm-suppress PossiblyNullPropertyFetch
+         */
+        if (
+            is_string($mediaId) &&
+            $caseData->idMethodIncludingNation->id_method === "PASSPORT" &&
```

Comment:
Can use enum here rather than passing the string around:
```suggestion
            $caseData->idMethodIncludingNation->id_method === IdMethod::PassportNumber &&
```